viewing_party/party.py

```diff
@@ -1,23 +1,123 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+def create_movie(title, genre, rating): 
+    if title == None or genre == None or rating == None:
+        return None
+    return {"title" : title, "genre" : genre, "rating" : rating}
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data

```

Comment:
What happens here if the user is trying to add a movie to their `watched_list` that is already there? Maybe we could implement some kind of guard clause?